### ISSUE TITLE
Installation fails (Laravel 5.2)

### ISSUE THREAD

**Issue by Mastergalen** [spatie/laravel-backup#61]
```
vagrant@homestead:~/Code/Zugy$ composer require spatie/laravel-backup
Using version ^2.9 for spatie/laravel-backup
./composer.json has been updated
> php artisan clear-compiled
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for spatie/laravel-backup ^2.9 -> satisfiable by spatie/laravel-backup[2.9.0].
    - Conclusion: remove symfony/finder v3.0.1
    - Conclusion: don't install symfony/finder v3.0.1
    - spatie/laravel-backup 2.9.0 requires symfony/finder ~2.6 -> satisfiable by symfony/finder[v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.2, v2.6.3, v2.6.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.8.0, v2.8.1].
    - Can only install one of: symfony/finder[v2.8.0, v3.0.1].
    - Can only install one of: symfony/finder[v2.8.1, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.0, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.1, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.10, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.11, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.12, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.2, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.3, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.4, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.5, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.6, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.7, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.8, v3.0.1].
    - Can only install one of: symfony/finder[v2.6.9, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.0, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.1, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.2, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.3, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.4, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.5, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.6, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.7, v3.0.1].
    - Can only install one of: symfony/finder[v2.7.8, v3.0.1].
    - Installation request for symfony/finder == 3.0.1.0 -> satisfiable by symfony/finder[v3.0.1].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by kyleh** [comment]
composer.json "require":

Laravel 5.0 - "symfony/finder": "2.6.*"
Laravel 5.1 - "symfony/finder": "2.7.*",
Laravel 5.2 - "symfony/finder": "2.8.*|3.0.*"

laravel-backup - "symfony/finder": "~2.6"

**Comment by freekmurze** [comment]
Fixed in v2.9.2
